apis/metal3.io/v1alpha1/bmceventsubscription_webhook.go

```diff
@@ -15,7 +15,7 @@ package v1alpha1
 import (
 	"fmt"
 
-	"k8s.io/apimachinery/pkg/util/errors"
+	kerrors "k8s.io/apimachinery/pkg/util/errors"
```

Comment:
This is named `k8serrors` elsewhere, maybe use the same?

Reply:
that is different 
```
      alias: k8serrors
    - pkg: k8s.io/apimachinery/pkg/util/errors
      alias: kerrors
    - pkg: k8s.io/component-base/logs/api/v1
```